src/js/ripple/amount.js

```diff
@@ -1,53 +1,55 @@
 // Represent Ripple amounts and currencies.
 // - Numbers in hex are big-endian.
 
-var extend = require('extend');
-var utils = require('./utils');
-var sjcl  = utils.sjcl;
-var bn    = sjcl.bn;
-
-var BigInteger = utils.jsbn.BigInteger;
-
-var UInt160  = require('./uint160').UInt160;
-var Seed     = require('./seed').Seed;
-var Currency = require('./currency').Currency;
-
-//
-// Amount class in the style of Java's BigInteger class
-// http://docs.oracle.com/javase/1.3/docs/api/java/math/BigInteger.html
-//
+var assert    = require('assert');
+var BigNumber = require('bignumber.js');
+var extend    = require('extend');
+var utils     = require('./utils');
+var UInt160   = require('./uint160').UInt160;
+var Seed      = require('./seed').Seed;
+var Currency  = require('./currency').Currency;
+
+utils.configureBigNumber(BigNumber);
```

Comment:
I don't like this structure. If I understand correctly, anyone requiring the `bignumber.js` package can override the configuration. I'd rather have a separate class that wraps the `bignumer.js` package, configures it and exposes a singleton for usage across the other source files.